images/bootstrap/Dockerfile

```diff
@@ -22,6 +22,9 @@ RUN mkdir -p /workspace
 ENV WORKSPACE=/workspace \
     TERM=xterm
 
+# Cleanup unnecessary package(s)
+RUN echo 'Yes, do as I say!' | apt-get remove systemd
```

Comment:
this is pretty much the best line of bash in `test-infra`.

Reply:
I'm moderately certain now that something below depends on systemd. If I `apt list --installed` in the built image I see:
```
systemd/oldstable,now 215-17+deb8u7 amd64 [installed,automatic]
systemd-sysv/oldstable,now 215-17+deb8u7 amd64 [installed,automatic]
```